### ISSUE TITLE
Cavernous sinus enhancement

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11632]
**Preferred term label:**
Cavernous sinus enhancement

**Synonyms**

**Definition (free text, please give PubMed ID)**
Similar to HP:6000851 but for the cavernous sinus

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tolosa Hunt Syndrome

**Your nano-attribution (ORCID)**
